x/datadeal/abci_test.go

```diff
@@ -268,10 +276,14 @@ func (suite abciTestSuite) TestDataVerificationEndBlockerVoteRejectSamePower() {
 		VerifiableCid: suite.verifiableCID,
 		DeliveredCid:  "",
 		Status:        types.DATA_SALE_STATUS_VERIFICATION_VOTING_PERIOD,
-		VotingPeriod: &oracletypes.VotingPeriod{
+		VerificationVotingPeriod: &oracletypes.VotingPeriod{
 			VotingStartTime: time.Now().Add(-2 * time.Second),
 			VotingEndTime:   time.Now().Add(-1 * time.Second),
 		},
+		DeliveryVotingPeriod: &oracletypes.VotingPeriod{
+			VotingStartTime: time.Now().Add(1 * time.Second),
+			VotingEndTime:   time.Now().Add(2 * time.Second),
+		},
```

Comment:
Don't you need this part?
DeliveryVotingPeriod is set a `EndBlocker`

Reply:
Yes, the `DeliveryVotingPeriod` set from voting period in `EndBlocker`. However, the test works well if the `DeliveryVotingPeriod` set in `DataVerificationVote`.

Reply:
Actually, the test must be passed if the `DeliveryVotingPeriod` has `nil` value. So I changed the value to `nil` in `DeliveryVotingPeriod` and the tests were passed.